go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/google/go-cmp v0.5.6
 	github.com/gorilla/websocket v1.4.2
-	github.com/lucas-clemente/quic-go v0.24.0
+	github.com/lucas-clemente/quic-go v0.24.1-0.20211216064023-1f3350557e2a
```

Comment:
I suggest let's wait for quic-go to formally release its new version before we update dependency